packages/create-pantheon-decoupled-kit/package.json

```diff
@@ -0,0 +1,65 @@
+{
+	"name": "create-pantheon-decoupled-kit",
```

Comment:
My understanding of `npm-init` is that the package name must start with `create-`. Maybe because we have the `create-pantheon-decoupled-kit` property in the `bin` object, we can change the name? I'll double check.